src/plugins/discourse/mirror.js

```diff
@@ -214,7 +215,8 @@ export class Mirror implements DiscourseData {
           author_username,
           topic_id,
           index_within_topic,
-          reply_to_post_index
+          reply_to_post_index,
+          cooked
         FROM posts`
       )
       .all()
```

Comment:
How does having `.all()` with the cooked HTML impact the memory footprint? Will all data be loaded into memory at once?

Reply:
Yeah, it loads all the posts into memory at once. When/if this becomes a memory issue we can address it by e.g. having an iterator over the posts so that they don't all need to be in memory at the same time. At the moment the full scraped db is only 1.9M so we're a long way from needing to worry about the memory footprint.